simpinkscr/simple_inkscape_scripting.py

```diff
@@ -3724,7 +3724,8 @@ def effect(self):
 
         # Remove an unnecessary import that may be introduced when
         # running from Visual Studio Code.
-        code.replace("from simpinkscr import *", "")
+        pattern = r"^(import (inkex|simpinkscr)|(from (inkex|simpinkscr) import.*))\r*\n*"
+        code = re.sub(pattern, "", code, flags=re.MULTILINE+re.IGNORECASE)
```

Comment:
What's the purpose of `re.IGNORECASE` here?

Reply:
Honestly, I think I've been looking at too much js.  I'm relatively new to Python, so wasn't sure if the import statements were case sensitive or not.  I'll remove it since it serves no purpose.